### ISSUE TITLE
Too few materials usable for Bow or Crossbow limbs

### ISSUE THREAD

**Issue by Secretname2009** [SilentChaos512/Silent-Gear#513]
## What Should Be Added or Changed?
Rebalance most materials for bow and crossbow limbs.
- (Description)
Rebalance most materials with focus on ranged damage and speed so that there are more options.
## Why Should This Be Done?

Much better selection and options for materials usable to make bow and crossbow limbs
- (Description)
I encountered this problem as i wanted to make a silentgear bow. It is severely needed as most materials are a bit nerfed or even downright unusable. A good example is wood which for some reason has a Ranged Damage and Speed value of 0x which makes it unusable for limbs. That is one of many other materials that should work as bow limbs but is just left at default which is set at a multiplier of 0. The only fairly effective materials with little negatives are Crimson Steel and Titanium.

**Comment by Secretname2009** [comment]
Just so its noted, im not sure if this was intended or not so i posted this as enhancement and not a bug. Correct me if im wrong.

**Comment by YeOldeDog** [comment]
I agree with the need for a rebalance. I made a Silent Gear bow earlier and was surprised when it performed worse than a comparatively cheaper Vanilla bow.

This is what I used: 
bow: wood bow limb, iron tool rod, string bowstring, leather grip
arrow: stone arrow head, wood tool rod, paper fletching
I fired more than 10 arrows at a Creeper and it didn't die, VS only needing to fire 3 with Vanilla bow & arrow to kill a Creeper.
With the Silent Gear bow & arrow I made it also took 6 arrows to kill a chicken.

I'm currently playing the modpack "Better Than Minecolonies Getting Started (BTMCGS)" 3.3.4 (current) which has silent-gear-1.17.1-2.7.10.jar installed.
